### ISSUE TITLE
clickable area  for search buttons too small

### ISSUE THREAD

**Issue by userquin** [vuejs/vitepress#2225]
### Describe the bug

The buttons at least on local search are small, and no space between them.

### Reproduction

Open UnocssCSS and search something, icons for example: https://unocss.dev

![imagen](https://user-images.githubusercontent.com/6311119/232252220-53824212-9462-4013-9e31-595d2a00e0df.png)

### Expected behavior

Search buttons should have 36px or 40px to allow click them.

### System Info

```shell
NA
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.dev).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by brc-dd** [comment]
fixed in 4449867
